environs/config/config.go

```diff
@@ -1706,4 +1718,9 @@ data of the store. (default false)`,
 		Type:        environschema.Tstring,
 		Group:       environschema.EnvironGroup,
 	},
+	BackupDirKey: {
+		Description: "Backup directory for temporal files",
```

Comment:
"Directory used to store temporal files when creating a backup"

Reply:
Updated.